docs/walkthroughs/developing-apbs-locally.adoc

```diff
@@ -46,6 +46,95 @@ apb init
 
 NOTE: Best practice is to name the APB ending with `-apb`.
 
+== Making your APB consumable as a mobile service
+
+When creating an APB for a mobile service, where possible use Kubernetes resources
+( rather than Openshift resources ) to ensure compatibility with Openshift and Kubernetes, eg. using 
+Kubernetes Deployment instead of openshift DeploymentConfig.
+
+
+Only one instance of a service is expected in a namespace. If your service is intended to have more
+than one service instance consideration must be given to the naming of objects.
+
+=== ConfigMaps Vs Secrets
+
+Separate the public and private configuration information in your APB.
+
+Use *configMaps* for public information that a mobile client will need to use the service. To allow the configMap
+to be read by the mobile-cli it must be labeled with *mobile: enabled* and the *serviceName*.
+
+An example of this can be seen link:https://github.com/aerogearcatalog/keycloak-apb/blob/master/roles/provision-keycloak-apb/templates/configmap.yml.j2[here].
+
+[source,yaml]
+----
+  labels:
+    mobile: enabled
+    serviceName: my-custom-service
+----
+
+It's possible that a mobile client will require some secret information such as an API key. This should be stored in the configMap.
+
+Use *secrets* for private information such as passwords and usernames.
+
+[source,yaml]
+----
+- k8s_v1_secret:
+    name: '{{ myService_secret_name }}'
+    namespace: '{{ namespace }}'
+    labels:
+      name: myService
+      mobile: enabled
+      serviceName: myService
+    string_data:
+      admin_username: '{{ ADMIN_NAME }}'
+      admin_password: '{{ ADMIN_PASSWORD }}'
+----
+
+For more information about this requirement see the original proposal link:https://github.com/aerogear/proposals/blob/master/apbs/create-secret-and-configmap-during-provision.md[here].
+
+=== Using the mobile-service tag
+
+For MCP enabled _APBs_, make sure there is a `mobile-service` tag inside the `apb.yml` file:
+
+....
+...
+tags: 
+  - mobile-service
+....
+
+This tag is used by the serviceCatalog to group the APB with other mobile services.
+
+=== Using the servicename label
+
+Inside of the `apb.yml` file, make sure you use the `serviceName:` label, like:
```

Comment:
Are those really labels (do they end up as labels on the service instance) or just metadata? Sorry for being pedantic but a `label` has a pretty specific meaning and readers could confuse the concepts.